client/app/lib/components/redux-form/SingleFileInput.jsx

```diff
@@ -70,6 +70,8 @@ class SingleFileInput extends React.Component {
       onChange: PropTypes.func.isRequired,
     }),
     intl: intlShape.isRequired,
+    isDropFileSupported: PropTypes.func,
```

Comment:
Possible to add the documentation for `isDropFileSupported` at the top of the document?

Reply:
OK, will add this.

Reply:
How about we name this `validateFileType`? I think that's less confusing and what this method does is actually validation.